modules/nodes-pods-image-volume-adding.adoc

```diff
@@ -29,18 +34,15 @@ spec:
   volumes:
   - name: volume 
     image: <1>
-      reference: quay.io/crio/artifact:v2 <2>
+      reference: quay.io/crio/image:v2 <2>
```

Comment:
Here and Line 75, use the reference "quay.io/crio/image:v2" , but in verification part, use the reference "quay.io/crio/volume:v2"(Line 108, 115, 116). 
Please keep consistency